pkg/placement/membership.go

```diff
@@ -362,7 +387,7 @@ func (p *Service) performTableDissemination(ctx context.Context) error {
 // It first locks so no further dapr can be taken it. Once placement table is locked
 // in runtime, it proceeds to update new table to Dapr runtimes and then unlock
 // once all runtimes have been updated.
-func (p *Service) performTablesUpdate(ctx context.Context, hosts []placementGRPCStream, newTable *v1pb.PlacementTables) error {
+func (p *Service) performTablesUpdate(ctx context.Context, hosts []placementGRPCStream, newTable *v1pb.PlacementTables, newTableWithVNodes *v1pb.PlacementTables) error {
```

Comment:
Parsing two parameters of the same type can be tricky for the caller to get right. Please can we use and pass a `tablesUpdate` struct

Reply:
Oh, I agree, this is ugly, and I'm actually considering a struct too, but I'm torn between having the code cleaner now vs. having these changes easier/obvious to revert when we do them in 1.15. What do you think?

Reply:
I think it is better to do the struct now. We’ll keep the struct forever so it would only be a case of removing the field in future. Using a struct will also move the need for `hasNewDapr` and `hasOldDaprds` above. Not a massive fan of `old` and `new` anywho as that doesn’t mean too much.